pkg/injector/injector.go

```diff
@@ -224,6 +234,7 @@ func (i *injector) handleRequest(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+SkipPatch:
```

Comment:
I really do not like any `goto` codes, could we use other way to do it?

Reply:
I'll change it.